sql/core/src/main/scala/org/apache/spark/sql/execution/SparkStrategies.scala

```diff
@@ -267,12 +267,32 @@ private[sql] abstract class SparkStrategies extends QueryPlanner[SparkPlan] {
   }
 
   object CartesianProduct extends Strategy {
+    def createCartesianProduct(left: LogicalPlan, right: LogicalPlan): SparkPlan = {
+      // If plan can broadcast we use BroadcastNestedLoopJoin, as we know for inner join with true
+      // condition is same as Cartesian.
+      // For CartesianProduct we will use the small size plan as cartesian left rdd.
+      if (right.statistics.sizeInBytes <= left.statistics.sizeInBytes) {
+        right match {
+          case CanBroadcast(right) => joins.BroadcastNestedLoopJoin(planLater(left),
```

Comment:
the line wrapping here is off.  if you have to wrap, don't leave anything on the line of the case statement.  if you have to wrap one parameter, wrap them all.